### ISSUE TITLE
how to write JSON to influxdb cloud using python

### ISSUE THREAD

**Issue by Mogoatlhe** [influxdata/influxdb-client-python#103]
Hi, i am having trouble sending data to my influxdb cloud instance in JSON format using Python 3.6.9.

`body =  Point("my_measurement").tag("location", "Prague").tag("location", "Tladistad").field("temperature", 25.3)
write_api.write(bucket=buck, org=organisation, record=body)`

the above works fine, but i want to use json (dictionary) and i also want have multiple tags and fields like below

`json_body =  [{"measurement": organisation, "tags":{"tag1":"tag1", "tag2": "tag2"}, "fields":{'field1':1, "field2":2}}]
write_api.write(bucket=buck, org=organisation, record=json_body)`

i get the error below

`Traceback (most recent call last):                                                                                                                                        File "initialize.py", line 12, in <module>                                                                                                                                write_api.write(bucket=buck, org=organisation, record=json_body)                                                                                                      File "/home/moeketsi/.local/lib/python3.6/site-packages/influxdb_client/client/write_api.py", line 217, in write                                                          final_string = self._serialize(record, write_precision, **kwargs)                                                                                                     File "/home/moeketsi/.local/lib/python3.6/site-packages/influxdb_client/client/write_api.py", line 263, in _serialize                                                     **kwargs) for item in record])                                                                                                                                        File "/home/moeketsi/.local/lib/python3.6/site-packages/influxdb_client/client/write_api.py", line 263, in <listcomp>                                                     **kwargs) for item in record])                                                                                                                                        File "/home/moeketsi/.local/lib/python3.6/site-packages/influxdb_client/client/write_api.py", line 253, in _serialize                                                     _result = self._serialize(Point.from_dict(record, write_precision=write_precision),                                                                                   File "/home/moeketsi/.local/lib/python3.6/site-packages/influxdb_client/client/write/point.py", line 37, in from_dict                                                     point.time(dictionary['time'], write_precision=write_precision)                                                                                                     KeyError: 'time'`

can you kindly assist, thank you (edited)

**Comment by bednar** [comment]
Hi @Mogoatlhe,

It was caused by bug in: https://github.com/influxdata/influxdb-client-python/blob/ab189350afbb3b4863cd4631d6ab80e12a7dc5b8/influxdb_client/client/write/point.py#L31

The issue is fixed in [1.8.0 milestone](https://github.com/influxdata/influxdb-client-python/milestone/10). 

As workaround you could use something like:

```python
json_body =  [{"measurement": organisation, "tags":{"tag1":"tag1", "tag2": "tag2"}, "fields":{'field1':1, "field2":2}, "time": None}]
write_api.write(bucket=buck, org=organisation, record=json_body)
```

or if you would like to use a dev version then install client via:

```pip install git+https://github.com/influxdata/influxdb-client-python.git@master```

Regards

**Comment by Mogoatlhe** [comment]
Thank you.
